baseplate_py_upgrader/fixes/v1_0/__init__.py

```diff
@@ -138,7 +138,7 @@ def update(
     refactor_python_files(root, __name__)
 
     wheelhouse.ensure(requirements_file, "cassandra-driver>=3.13.0")
-    wheelhouse.ensure(requirements_file, "cqlmapper>=0.1.0")
+    wheelhouse.ensure(requirements_file, "cqlmapper>=0.2.0")
```

Comment:
```suggestion
    wheelhouse.ensure(requirements_file, "cqlmapper>=0.2.2")
```

Ok if we go to `0.2.2`? That was after we added `load_many` and stopped using `select *` in it.

Reply:
This is just a minimum requirement. If they're already on 0.2 of any flavor it'll bail out but if they're below 0.2 they'll get whatever is the latest option in the wheelhouse.